components/app-proxy/vm.tf

```diff
@@ -6,78 +6,65 @@ module "tags" {
   builtFrom   = var.builtFrom
 }
 
-resource "azurerm_resource_group" "this" {
+resource "azurerm_resource_group" "app_proxy_rg" {
   name     = "${var.product}-${var.env}-rg"
   location = var.location
 
   tags = module.tags.common_tags
 }
 
-# TODO store in a vault
-resource "random_password" "this" {
-  length = 30
-}
-
-resource "azurerm_subnet" "this" {
-  name = "app-proxy"
-
+resource "azurerm_subnet" "app_proxy_subnet" {
+  name                 = "app-proxy"
   address_prefixes     = ["10.99.73.0/25"]
-  resource_group_name  = "mgmt-vpn-2-mgmt"
-  virtual_network_name = "mgmt-vpn-2-vnet"
+  resource_group_name  = var.app_proxy_subnet_rg
+  virtual_network_name = var.app_proxy_vnet_name
 }
 
 # TODO auto-register VMs in DNS with private dns autoregistration
```

Comment:
is this done? or could we do it? should be easy

Reply:
It's not done here, I'll take a look though